pkg/resources/validation/cluster/cluster.go

```diff
@@ -270,15 +259,3 @@ func (v *Validator) validatePSP(request *admission.Request) (*admissionv1.Admiss
 
 	return admission.ResponseAllowed(), nil
 }
-
-func getClusterVersion(version string) (semver.Version, error) {
```

Comment:
Note to reviewer: 
Code refactor, this function is moved to `pkg/podsecurityadmission/podsecurityadmission.go`